operate/services/manage.py

```diff
@@ -607,6 +608,12 @@ def _deploy_service_onchain_from_safe(  # pylint: disable=too-many-statements,to
             self._get_on_chain_state(service=service, chain=chain)
             == OnChainState.NON_EXISTENT
         )
+
+        # TODO Determine if the mint metadata hash changed - if so, it requires
+        # an on-chain update.
+        # This has to be implemented. Temporarily added as a placeholder (ignored).
+        has_metadata_changed = False
```

Comment:
@OjusWiZard This is missing to be implemented. The idea is that if there is a change in the metadata (mainly service.description) compared with the metadata minted on-chain, this flag should be true. This will allow to update the new metadata with the new Twitter handle (e.g. for Memeooorr). 

This value has to be false if the service is not yet minted.

Reply:
This is not planned in the MVP, but we have noted it for the future